UofS-Pulse-Binfo/rawphenotypes
# Issue #27 - Search new stock name, ignore column and email support.

NOTE
          1. this update requires an update to KP nodes to include all
hook_alter() for this update to apply (Issue #6 - Rawphenotypes hook_alter() implementations).
          2. Should update code when - Issue #12 Stock names look-up should be restricted to an organism is merged.

- Upload functionality can now search new name (using a token eg. AGL)
to a specified project name.
- Ignore or skip a set column names.
- Provide a support email for data collectors to email for support.

To test:
A. Support email:
  In the homepage of you dev, in rawphenotypes dashboard, click need help? link and test 
  1. help topic select jump menu and 
  2. support email (please click if it starts a blank email or right click on the link to copy the email address to manually compose an email)

B. Origin
  1. In standard spreadsheet file, add a column Origin.
  2. Upload the spreadsheet file. Ensure to upload file into AGILE project.
     In all upload stages, it should skip this column.
  3. When done, download dataset to verify.

C. New stock names (with AGL).
  1. Upload to AGILE project a spreadsheet file where the name column contains stock names without AGL token.
  2. When complete, download dataset to verify the use of new stock names.
